app/templates/components/conversations/conversation-part-closed.hbs

```diff
@@ -0,0 +1,13 @@
+<div class="coversation-part-closed__message-wrapper">
+  <div class="conversation-part-closed__timestamps-wrapper">
+    <div class="conversation-part-closed__timestamps">
+      <span data-test-closed-at>
+  {{#if isSelf}}
+    You closed this {{moment-from-now closedAt interval=60000}}
+  {{else}}
+    {{author.username}} closed this {{moment-from-now closedAt interval=60000}}
+  {{/if}}
+      </span>
+    </div>
+  </div>
+</div>
```

Comment:
In line with my comment above, you can remove all the `-closed` parts from all the classes here.